antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -472,7 +545,74 @@ def copy(
             with_outputs: indicate either to copy the output or not
         Returns: destination study
         """
-        raise NotImplementedError()
+
```

Comment:
(ignore the next comments)
this is not how it's intended to work.
it should only copy the variant's commands and patch.json (and the outputs if needed)